mcs/class/System.Private.CoreLib/Internal.Runtime.Augments/RuntimeThread.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Threading;
+using System.Runtime.ConstrainedExecution;
+using System.Runtime.CompilerServices;
+
+namespace Internal.Runtime.Augments
+{
+	public partial class RuntimeThread : CriticalFinalizerObject
+	{
+		// Note: Magic number copied from CoreRT's RuntimeThread.cs. See the original source code for an explanation.
+		internal static readonly int OptimalMaxSpinWaitsPerSpinIteration = 64;
+
+		readonly Thread thread;
+
+		RuntimeThread (Thread t) { thread = t; }
+
+		private Thread AsThread()
+		{
```

Comment:
Shouldn't these files use the mono coding conventions ?

Reply:
I am not sure, I suspect some changes will eventually migrate to shared which means coreclr coding convention, do we want to do the conversion at that point?